### ISSUE TITLE
manual/scalaGuide/main/http/ScalaActions.md

### ISSUE THREAD

**Issue by garbagetown** [playframework-ja/translation-project#1005]
- https://github.com/playframework-ja/translation-project/tree/2.3.5/manual/scalaGuide/main/http/ScalaActions.md
- diffs between [2.2.0/manual/scalaGuide/main/http/ScalaActions.md](https://github.com/playframework-ja/translation-project/tree/2.2.0/manual/scalaGuide/main/http/ScalaActions.md) and [2.3.5/manual/scalaGuide/main/http/ScalaActions.md](https://github.com/playframework-ja/translation-project/tree/2.3.5/manual/scalaGuide/main/http/ScalaActions.md).

```
--- /Users/garbagetown/Desktop/2.2.0/manual/scalaGuide/main/http/ScalaActions.md    2013-09-19 10:11:22.000000000 +0900
+++ //Users/garbagetown/Desktop/2.3.5/manual/scalaGuide/main/http/ScalaActions.md   2014-07-28 12:26:38.000000000 +0900
@@ -1,3 +1,4 @@
+<!--- Copyright (C) 2009-2013 Typesafe Inc. <http://www.typesafe.com> -->
 # Actions, Controllers and Results

 ## What is an Action?
@@ -50,7 +51,7 @@

 For now we are just interested in simple results: An HTTP result with a status code, a set of HTTP headers and a body to be sent to the web client.

-These results are defined by `play.api.mvc.SimpleResult`:
+These results are defined by `play.api.mvc.Result`:

 @[simple-result-action](code/ScalaActions.scala)


```

**Comment by garbagetown** [comment]
An untranslated line exists in "Controllers are action generators" block. This problem may exist in old documents, too.

**Comment by garbagetown** [comment]
:hand:

**Comment by garbagetown** [comment]
merged at #1061
